tombstones/tombstones.go

```diff
@@ -54,8 +71,8 @@ type TombstoneReader interface {
 	Close() error
 }
 
-func writeTombstoneFile(logger log.Logger, dir string, tr TombstoneReader) (int64, error) {
-	path := filepath.Join(dir, tombstoneFilename)
+func WriteTombstoneFile(logger log.Logger, dir string, tr TombstoneReader) (int64, error) {
```

Comment:
As it is exposed we need comment e.g for goDoc ): Same everywhere else for public functions we add.